packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -568,9 +573,10 @@ export async function handleSocialLoginCallback({
     // if email is not set, set the SSO email as the default email
     // only if its a standalone account (no account linking)
     if (!userStore.getState().email && ssoEmail && profiles?.length === 1) {
-      await updateEmail({ email: ssoEmail })
-        .then(onUpdateEmailSuccess)
-        .catch(() => onUpdateEmailError(false));
+      // TODO: what should we do here? - @masvelio
```

Comment:
as this is non-react context, we need to use axios calling trpc endpoint. We use this pattern in other places as well. You did this recently 

![image](https://github.com/user-attachments/assets/ab51be20-e224-49f6-ae1d-bea724bdb83b)